src/System.Runtime.Serialization.Xml/tests/DataContractSerializer.cs

```diff
@@ -14,6 +14,8 @@
 using SerializationTypes;
 
 using Xunit;
+using System.Threading.Tasks;
+using System.Threading;
```

Comment:
Nice catch. The new tests were added in this file. I forgot to remove these usings when I moved the tests to a new file.